k8s/lke/media/patch-pv.yaml

```
@@ -0,0 +1,21 @@
+#
+# Nimbus
+# Media Streaming Service
+# Persistent Volume Patch
+#
+
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: media-data
+spec:
+  csi:
+    volumeAttributes:
+      remotePath: mrzzy-co-media/csi-fs
+      # both jellyfin & rtorrent-flood pod's have a emptydir volume mounted at /cache
+      cache-dir: /cache
+      # enable full vfs cache mode to cache media files while jellyfin is streaming them
```

```suggestion
      # enable full read / write vfs cache mode to cache rtorrent-flood's writes & jellyfin's reads
```
Document which deployment is using read or write VFS caching modes..